### DISCUSSION TITLE
Flipt CLI: Import issue when --address and --drop is used

### DISCUSSION THREAD

**Discussion by jordi-gou-flyr** [flipt-io/flipt#3221] [Q&A]
I have a Flipt running in a minikube which is accessible through http://localhost:8000 in my computer.

There is no content in Flipt at the beginning
```
$ flipt export --address http://localhost:8000

version: "1.2"
namespace: default
```
Then, I created a yaml file to import content there
```
$ cat init-state.yaml

version: "1.2"
namespace: default
flags:
- key: my-first-flag
  name: my-first-flag
  type: VARIANT_FLAG_TYPE
  enabled: false
```

I'm able to import this data there

```
$ flipt import --address http://localhost:8000 init-state.yaml

$ flipt export --address http://localhost:8000

version: "1.2"
namespace: default
flags:
- key: my-first-flag
  name: my-first-flag
  type: VARIANT_FLAG_TYPE
  enabled: false  
```

However, if I try to import that yaml file again, I get an error because of the conflicts. This is normal

```
$ flipt import --address http://localhost:8000 init-state.yaml

Error: creating flag: rpc error: code = InvalidArgument desc = flag "default/my-first-flag" is not unique
```

For that reason, I added the --drop flag to the import. But, it seems it is doing nothing

```
$ flipt import --address http://localhost:8000 --drop init-state.yaml

Error: creating flag: rpc error: code = InvalidArgument desc = flag "default/my-first-flag" is not unique
```

Could you tell me what I'm doing wrong?

Thanks

**Comment by GeorgeMac** [comment]
Hey @jordi-gou-flyr 

Apologies, the `--drop` flag does only work with the non `--address` form, which goes direct to the database and skips the API.
We need to put in a guard to explain that in the near term.

Then maybe we could explore adding support for `--drop` with `--address`.

**Reply by GeorgeMac** [reply]
I spun this off as an issue we can use to track the tidy up for this https://github.com/flipt-io/flipt/issues/3222 👍
